fury/utils.py

```diff
@@ -1,3 +1,4 @@
+from nibabel.streamlines.array_sequence import ArraySequence
```

Comment:
FURY does not depend on Nibabel. Other fields do not need `nibabel` and we want to stay the most generic as possible for FURY project.

Reply:
How do I check if it's an ArraySeqence then?
I test if not a list?